package.json

```diff
@@ -2,7 +2,12 @@
 	"name": "@adonisjs/core",
 	"version": "5.1.8",
 	"description": "Core of AdonisJS",
-	"main": "build/providers/AppProvider.js",
+	"exports": {
+		".": "./build/providers/AppProvider.js",
+		"./standalone": "./build/standalone.js",
+		"./Ignitor": "./build/src/Ignitor/index.js",
```

Comment:
happy to add more exports like this if there are other files meant to be imported by the end user.

Reply:
Does it work with CJS too?

Reply:
Yes it does.

Reply:
```
> require('@adonisjs/core/Ignitor') == require('@adonisjs/core/build/src/Ignitor/index.js')
true
```